bin/iron/src/app.rs

```diff
@@ -127,6 +127,11 @@ async fn init(app: &tauri::App, args: &Args) -> AppResult<()> {
     iron_networks::init(resource(app, "networks.json")).await;
     iron_forge::init().await?;
 
+    
+    if let Err(err)= tokenlist::init().await {
```

Comment:
I don't think this should be here. as I said, the list should be built at compile time (in a `build.rs` script). don't think we need to init something here at runtime